### DISCUSSION TITLE
What is the current best way to see which version of observable quarto is using?

### DISCUSSION THREAD

**Discussion by boshek** [quarto-dev/quarto-cli#6112] [Q&A] [answered]
### Description

This includes versions of the standard libraries like Plot, d3 etc. I've had to go through changelogs and discussions to find out this information and I am wondering if there is an easier way. If there is not...

I am wondering if that would be worth including in `quarto check` if it isn't already easily accessible.

**Comment by cscheid** [comment, marked as answer]
I don't think we want to provide all of that information on every run of `quarto check`, but it is probably a good idea to create a separate command that exposes the versions of the libraries we depend on.

In the case of ObservableJS specifically, you can get the version strings for their libraries directly via `Plot.version`.

**Reply by boshek** [reply]
Yeah a `quarto ojs_sitrep` would be great. It is sometimes hard to traverse, forum answers on observable, observablehq notebooks and quarto documentation to figure how to do a certain thing.
